libraries/api-extractor-model/src/mixins/ApiItemContainerMixin.ts

```diff
@@ -119,6 +121,7 @@ export function ApiItemContainerMixin<TBaseClass extends IApiItemConstructor>(
 
       this[_members] = [];
       this[_membersByContainerKey] = new Map<string, ApiItem>();
+      this.preserveEnumMemberOrder = options.preserveEnumMemberOrder ? options.preserveEnumMemberOrder : false ;
```

Comment:
Can be simplified to `this.shouldSortMembers = options.shouldSortMembers ?? true`.